code/style.css

```diff
@@ -1,58 +1,377 @@
-body {
+* {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  box-sizing: border-box;
+}
+
+/*** GENERAL ***/
+
+body {
+  font-family: 'Courier New', Courier, monospace;
+  background-color: #000;
 }
 
 p {
+  width: 100%;
   margin: 1px;
   font-weight: 300;
   background-color: #fff;
 }
 
-.cards {
-  margin: 50px auto;
+.visible {
+  display: flex;
+}
+
+.invisible {
+  display: none;
+}
+
+/*** HEADER WINDOW ***/
+
+.header-wrapper {
+  flex-direction: column;
+  justify-content: center;
+  height: 100vh;
+}
```

Comment:
I think we need to have display: flex before activating flex-direction and justify-content.